tests/tickets/16820/manual/ie10hrline.html

```diff
@@ -0,0 +1,14 @@
+<div id="editor">
+	<p>Foo</p>
+</div>
```

Comment:
I think the more readable formatting will be (newline separating editor html and js "logic"):

```
<div id="editor">
	<p>Foo</p>
</div>

<script>
	if ( !( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) ) {
        bender.ignore();
    }

	CKEDITOR.replace('editor', {
		height: 300,
		enterMode: CKEDITOR.ENTER_BR
	} );
</script>
```